src/main/java/com/esri/samples/featurelayers/list_releated_features/ListRelatedFeaturesSample.java

```diff
@@ -0,0 +1,167 @@
+
+      // add the map to the map view
+      mapView = new MapView();
+      mapView.setMap(map);
+
+      // wait until the map is done loading
+      map.addDoneLoadingListener(() -> {
+        // get the first feature layer for querying
+        FeatureLayer featureLayer = (FeatureLayer) map.getOperationalLayers().get(0);
+
+        // make selection outline yellow (0xFFFFFF00) and thick
+        featureLayer.setSelectionColor(0xFFFFFF00);
+        featureLayer.setSelectionWidth(5);
+
+        mapView.setOnMouseClicked(event -> {
+          // check for primary or secondary mouse click
+          if (event.isStillSincePress() && event.getButton() == MouseButton.PRIMARY) {
+
+            // create a point from where the user clicked
+            Point2D point = new Point2D(event.getX(), event.getY());
+
+            // convert to map coordinate
+            Point mapPoint = mapView.screenToLocation(point);
+
+            // identify the clicked features
+            QueryParameters queryParameters = new QueryParameters();
+            queryParameters.setGeometry(GeometryEngine.buffer(mapPoint, 10));
+            final ListenableFuture<FeatureQueryResult> selectFeatureQuery = featureLayer.selectFeaturesAsync
+                (queryParameters, FeatureLayer.SelectionMode.NEW);
+            selectFeatureQuery.addDoneListener(() -> {
+
+              try {
+                FeatureQueryResult result = selectFeatureQuery.get();
+                // get the first selected feature
+                Iterator<Feature> iterator = result.iterator();
+                if (iterator.hasNext()) {
+                  ArcGISFeature selectedFeature = (ArcGISFeature) iterator.next();
+                  // get the feature's feature table
+                  ArcGISFeatureTable featureTable = selectedFeature.getFeatureTable();
+
+                  // query related features
+                  final ListenableFuture<List<RelatedFeatureQueryResult>> relatedFeatureQuery = featureTable
+                      .queryRelatedFeaturesAsync(selectedFeature);
+                  relatedFeatureQuery.addDoneListener(() -> {
+                    try {
+                      //clear previous results
+                      accordion.getPanes().clear();
+                      // add all related features (grouped) in panes of the accordion
+                      List<RelatedFeatureQueryResult> results = relatedFeatureQuery.get();
+                      for(RelatedFeatureQueryResult relatedFeatureQueryResult : results){
+                        ListView<String> featureList = new ListView<>();
+                        String relatedTableName = relatedFeatureQueryResult.getRelatedTable().getTableName();
+                        // create a pane for the feature table with a list for its features
+                        TitledPane tablePane = new TitledPane(relatedTableName, featureList);
+                        accordion.getPanes().add(tablePane);
```

Comment:
Can we set the `Accordion` expanded by default? 

```java
accordion.setExpandedPane(tablePane);
```

Reply:
@doneill I've made it so that the last table (with the most features) is expanded by default.